metrics/grafana/tidb.json

```diff
@@ -9263,26 +9263,19 @@
           "stack": false,
           "steppedLine": false,
           "targets": [
-            {
-              "expr": "histogram_quantile(1, sum(rate(tidb_ddl_batch_add_idx_duration_seconds_bucket{tidb_cluster=\"$tidb_cluster\"}[1m])) by (le, type))",
```

Comment:
Why remove this？

Reply:
This PR only fix adding index rate metrics, maybe need to add worker duration in another PR. The rate and duration should in two separate Grafana panels.

Reply:
So do we need to remove `tidb_ddl_batch_add_idx_duration_seconds_bucket` in `infoschema/metric_table_def.go`?

Reply:
I don't think so. This metrics should be added back in the latter PR. However, the metrics adjustment requires changing the json file, and this kind of modification by adding a panel is not very controllable. So, I prefer to make a small change in this PR and add the panel in the next PR.

Reply:
Got it.